specs/refactor_common_code_and_move.adoc

```diff
@@ -203,4 +232,8 @@ None
 
 == References
 
-None
+* https://github.com/Tendrl/common/issues/58
+
+* https://github.com/Tendrl/common/issues/60
+
+* https://github.com/Tendrl/common/issues/62
```

Comment:
Have a mention about issue 60 as well in the document. That means common module specific logging and configuirations will be removed. This has a documentaion impact as well

Also you need to create individual issues per component(changes occuring) and link it here